modules/gdscript/gd_editor.cpp

```diff
@@ -44,19 +44,23 @@ void GDScriptLanguage::get_string_delimiters(List<String> *p_delimiters) const {
 	p_delimiters->push_back("' '");
 }
 Ref<Script> GDScriptLanguage::get_template(const String &p_class_name, const String &p_base_class_name) const {
+	bool use_space_indentation = EDITOR_DEF("text_editor/indent/type", "Tabs") == "Tabs" ? 0 : 1;
 
 	String _template = String() +
-					   "extends %BASE%\n\n" +
-					   "# class member variables go here, for example:\n" +
-					   "# var a = 2\n" +
-					   "# var b = \"textvar\"\n\n" +
-					   "func _ready():\n" +
-					   "\t# Called every time the node is added to the scene.\n" +
-					   "\t# Initialization here\n" +
-					   "\tpass\n";
+				    "extends %BASE%\n\n" +
```

Comment:
Ooooh my. Did you just use both spaces and tabs for aligning there?

I think it might be better if just a single tab (and no spaces) is used for those lines, but I would like to have a confirmation from @akien-mga on that